testing/base/testing_exe_path.h

```diff
@@ -2,8 +2,8 @@
 // Exceptions. See /LICENSE for license information.
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
-#ifndef CARBON_TESTING_BASE_GTEST_MAIN_H_
-#define CARBON_TESTING_BASE_GTEST_MAIN_H_
+#ifndef CARBON_TESTING_BASE_TESTING_EXE_PATH_H_
```

Comment:
Maybe rename this file to just `exe_path` since `testing` is already in the directory structure?

I mean, we do have common:exe_path, but I don't think that the name mirroring is a problem.

Reply:
The name mirroring is why I had the testing...

It also does something quite different -- this even depends on `//common:exe_path`... Is there maybe a better but different name?

Reply:
Going with `global_exe_path` which telegraphs the fact that this is really a global storage.